ui-speedspacechart/src/components/helpers/drawElements/speedLimitTags.ts

```diff
@@ -82,108 +85,135 @@ export const drawSpeedLimitTags = async ({
+        ctx.moveTo(x + textWidth + 2, Y_POSITION);
+        ctx.lineTo(x + textWidth + 2, Y_POSITION + RECT_HEIGHT);
         ctx.closePath();
+        ctx.stroke();
+      }
+
+      if (tag === 'incompatible' || tag === 'missing_from_train') {
+        const image = tag === 'incompatible' ? alertFillImage : questionImage;
+
+        ctx.fillStyle = color;
 
-        if (tag === 'UU') {
-          ctx.lineWidth = 0.5;
-          ctx.beginPath();
-          ctx.moveTo(x + textWidth + 2, Y_POSITION);
-          ctx.lineTo(x + textWidth + 2, Y_POSITION + RECT_HEIGHT);
-          ctx.closePath();
-          ctx.stroke();
-        }
-
-        if (tag === 'incompatible' || tag === 'missing_from_train') {
-          const image = tag === 'incompatible' ? alertFillImage : questionImage;
-
-          ctx.fillStyle = color;
-
-          const iconXPosition = x + (tagWidth - ICON_BACKGROUND_WIDTH) / 2;
-          const iconYPosition = Y_POSITION - ICON_OFFSET;
-          const cornerRadius = 4;
-
-          drawRoundedRect(
-            ctx,
-            iconXPosition,
-            iconYPosition,
-            ICON_BACKGROUND_WIDTH,
-            ICON_BACKGROUND_HEIGHT,
-            cornerRadius
-          );
-
-          drawSvgImageWithColor(
-            ctx,
-            image,
-            iconXPosition + ICON_OFFSET,
-            iconYPosition + ICON_OFFSET,
-            ICON_WIDTH,
-            ICON_HEIGHT,
-            '#FFFFFF'
-          );
-
-          if (tag === 'incompatible' && cursor.x && cursor.y) {
-            if (
-              cursor.x >= x - MARGIN_LEFT + leftOffset &&
-              cursor.x <= x - MARGIN_LEFT + leftOffset + tagWidth &&
-              cursor.y >= marginTop - MARGIN_TOP + Y_POSITION - 2 &&
-              cursor.y <= marginTop - MARGIN_TOP + Y_POSITION - 1 + RECT_HEIGHT
-            ) {
-              tooltip = {
-                cursorX: cursor.x + MARGIN_LEFT,
-                cursorY: Y_POSITION,
-                text: 'Incompatible with the infrastructure',
-              };
-            }
-          }
-        } else {
-          ctx.fillStyle = 'white';
-          ctx.font = '600 12px "IBM Plex Sans"';
-          ctx.textAlign = 'left';
-          ctx.textBaseline = 'middle';
-          const textPosition = x + TEXT_LEFT_PADDING;
-          ctx.fillText(tag, textPosition, Y_POSITION + TEXT_PADDING_TOP + RECT_HEIGHT / 2);
-        }
+        const iconXPosition = x + (tagWidth - ICON_BACKGROUND_WIDTH) / 2;
+        const iconYPosition = Y_POSITION - ICON_OFFSET;
+        const cornerRadius = 4;
+
+        drawRoundedRect(
+          ctx,
+          iconXPosition,
+          iconYPosition,
+          ICON_BACKGROUND_WIDTH,
+          ICON_BACKGROUND_HEIGHT,
+          cornerRadius
+        );
+
+        drawSvgImageWithColor(
+          ctx,
+          image,
+          iconXPosition + ICON_OFFSET,
+          iconYPosition + ICON_OFFSET,
+          ICON_WIDTH,
+          ICON_HEIGHT,
+          '#FFFFFF'
```

Comment:
nit: you could import { WHITE } from `const` then use it with `hex()` as in `reticle.ts`

Reply:
Will do!